### PR TITLE
Document application owners endpoint

### PR DESCRIPTION
**Pull request by valioprea**
- for the new API which retrieves owners details for a specific application

### CODE REVIEW

File: source/includes/_platform.md [hunk 1]
```diff
@@ -246,3 +246,44 @@ applicationId | true | Application ID to remove owner from
...
+    "userId": "00000000-0000-0000-0000-000000000000",
+    "email": "bear@doordeck.com",
+    "displayName": "Bear",
+    "orphaned": false,
+    "foreign": false
+  }
+]
+```
+- `displayName` is optional and will be null if not set
+- `orphaned` is optional and will be null if not set
```
Comment (mpbarnwell) [inline source/includes/_platform.md:289]: I don't recall this being optional, it should just be false if not orphaned.

**Review by mpbarnwell — APPROVED** [review]
I updated the PR title, this is a public repository so we need to try and be a little more descriptive and I've only added some minor comments so approved based on the assumption you'll fix those first.

File: source/includes/_platform.md [hunk 2]
```diff
@@ -246,3 +246,44 @@ applicationId | true | Application ID to remove owner from
...
+
+### HTTP Request
+
+`GET https://api.doordeck.com/platform/application/APPLICATION_ID/owner`
+
+### Request Parameters
+
+Parameter | Required | Description
+--------- | -------- | -----------
+applicationId | true | Application ID to get owner details for
```
[the remark this answers is not in the document]
Comment (valioprea) [reply source/includes/_platform.md:271]: Love it!

File: source/includes/_platform.md (hunk 1 above)
Comment (valioprea) [reply source/includes/_platform.md:289]: This is in solving by: https://github.com/doordeck/doordeck-core/issues/27

**Pull request merged**